client.go

```diff
@@ -52,3 +86,16 @@ func (c *Client) Move(seqset *imap.SeqSet, dest string) error {
 func (c *Client) UidMove(seqset *imap.SeqSet, dest string) error {
 	return c.move(true, seqset, dest)
 }
+
+// MoveWithFallback tries to move if the server supports it. If it doesn't, it
+// falls back to copy, store and expunge, as defined in section 3.3 of RFC6851.
```

Comment:
It's better to write "RFC 6851 section 3.3" because GoDoc recognizes it and creates a link for it

Reply:
Didn't knew that, nice feature by GoDoc. Should be fixed now.